views/employee.js

```diff
@@ -0,0 +1,35 @@
+const render = function(employee) {
+  return `
+  <div class="container employee-view">
+  <div class="row top">
+    <div class="col">
+    </div>
+  </div>
+  <div class="employee-table">
+    <div class="row align-items-center table-row name">
+      <div class="col col-sm-2 col-label name-label">
+        Name:
+      </div>
+      <div class="col col-data name-employee">
+        ${employee.first_name} ${employee.last_name}
+      </div>
+    </div>
+    <div class="row align-items-center table-row email">
+      <div class="col col-sm-2 col-label email-label">Email:</div>
+      <div class="col col-data email-employee">${employee.userId}</div>
+    </div>
```

Comment:
@AnthonyGarces you can see here that i'm just pulling in the employee foreign key (Id) here for the user email and below for the store id  - i believe it'll be pretty simple to associate email/store name using the Id but i didn't want to touch the models that nelan was working on